libvalkeycluster/vkutil.h

```diff
@@ -30,26 +30,26 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef __HIUTIL_H_
-#define __HIUTIL_H_
+#ifndef __VKUTIL_H_
+#define __VKUTIL_H_
 
 #include <stdint.h>
 #include <sys/types.h>
 
-#define HI_OK 0
-#define HI_ERROR -1
-#define HI_EAGAIN -2
+#define VK_OK 0
```

Comment:
Here I also got concerned about possible name clash with Vulkan symbols

Reply:
Issue resolved in other comment, lets keep VK